### PR TITLE
Extend Help Center and fix some warnings

### LINKED ISSUE #212: Help center: Update necessary
Many different PR have been merged recently. I have to update the new Help Center accordingly.

### PR DESCRIPTION
**Pull request by DL6ER**
Fixes #212 and some warning reported in the `error.log` file of `lighttpd`.

Changes proposed in this pull request:

- Extended `Help Center` to reflect the various additional features through the recent merges. It can and should now be updated with each modification that is proposed here

- Check for existence of `WEBPASSWORD` and `TEMPERATUREUNIT` before trying to access it to prevent the reporting of warnings.

- Remove unnecessary `start_session();` in queries.php leading to a PHP warning. It loads header.php which requires password.php which will always do a `session_start();`.

@Mcat12 Please review my changes to `help.php` for linguistic correctness.

@pi-hole/dashboard

### CODE REVIEW

File: help.php [hunk 1]
```diff
@@ -79,6 +94,12 @@
 </div>
 <div class="row">
     <div class="col-md-12">
+    <h2>Query adlists</h2>
+    <p>Runs the command <pre>sudo pihole -q <i>searchstring</i></pre> and prints the result transparently to the web UI. This command can be used to scan for strings in the list of blocked domains, e.g., &quot;analytics&quot; will deliver all domains which *analytics*, where the asterisk (*) represents a number of characters or an empty string</p>
```
Comment (Mcat12) [inline help.php]: will deliver all domains which contain "analytics".

**Comment by dschaper** [comment]
If this was a branch of the Pi-hole repo we'd be able to commit to it and do some of the updates ourselves. Since it's a forked branch we can't do much more than request the changes.

**Comment by DL6ER** [comment]
Ah, yes. Haven't thought about that. Will make it better next time!

**Comment by Mcat12** [comment]
Approved

**Pull request merged**